tests/contrib/langchain/test_langchain_llmobs.py

```diff
@@ -517,6 +578,52 @@ def test_llmobs_anthropic_chat_model(self, langchain_anthropic, mock_llmobs_span
         assert mock_llmobs_span_writer.enqueue.call_count == 1
         _assert_expected_llmobs_llm_span(span, mock_llmobs_span_writer, input_role="user")
 
+    @pytest.mark.skipif(sys.version_info < (3, 10, 0), reason="Requires unnecessary cassette file for Python 3.9")
+    def test_llmobs_embedding_query(self, langchain_core, langchain_openai, mock_llmobs_span_writer, mock_tracer):
+        embedding_model = langchain_openai.OpenAIEmbeddings()
+        trace = self._embed_query(
+            embedding_model=embedding_model,
+            query="hello world",
+            mock_tracer=mock_tracer,
+            cassette_name="openai_embedding_query.yaml",
+        )
+        assert mock_llmobs_span_writer.enqueue.call_count == 1
+        mock_llmobs_span_writer.enqueue.assert_called_with(
+            _expected_llmobs_llm_span_event(
+                trace[0],
+                span_kind="embedding",
+                model_name=embedding_model.model,
+                model_provider="openai",
+                input_documents=[{"text": "hello world"}],
+                output_value="[1 embedding(s) returned with size 1536]",
+                tags={"ml_app": "langchain_test"},
+                integration="langchain",
+            )
+        )
+
+    @pytest.mark.skipif(sys.version_info < (3, 10, 0), reason="Requires unnecessary cassette file for Python 3.9")
+    def test_llmobs_embedding_documents(self, langchain_core, langchain_openai, mock_llmobs_span_writer, mock_tracer):
+        embedding_model = langchain_openai.OpenAIEmbeddings()
+        trace = self._embed_documents(
+            embedding_model=embedding_model,
+            documents=["hello world", "goodbye world"],
+            mock_tracer=mock_tracer,
+            cassette_name="openai_embedding_document.yaml",  # cassette doesn't exist on langchain directory
```

Comment:
If you are using FakeEmbeddings as a test case, you will most likely not need a cassette as that is local